rosa_backing_up_and_restoring_applications/backing-up-applications.adoc

```diff
@@ -6,6 +6,12 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can employ OpenShift API for Data Protection (OADP) with Red Hat OpenShift Service on AWS (ROSA) clusters to backup and restore application data. A ROSA deployment of OpenShift is configured specifically for AWS services.
+You can employ OpenShift API for Data Protection (OADP) with Red Hat OpenShift Service on AWS (ROSA) clusters to backup and restore application data. To do so, you need to set up role and policy credentials for OADP so that it can use the AWS API. 
 
-include::modules/oadp-installing-oadp-rosa-sts.adoc[leveloffset=+1]
\ No newline at end of file
+This is a two stage process:
```

Comment:
I'd argue that this part isn't truly necessary. You've already mentioned above that before installing OADP, you have to set up the credentials

If you do keep it, I'd consider rewording the sentence "This is a two stage process:". I don't have a great suggestion to give at the moment, but I think the "two stage" part could have better wording, and it might not be super clear what "This" is.

Reply:
@arendej  and @weshayutin do I leave wording at "THis is a two stage process" ?

Reply:
It makes enough sense to me, let's not bikeshed.